src/MainWindow.vala

```diff
@@ -186,7 +186,7 @@ public class SnippetPixie.MainWindow : Gtk.ApplicationWindow {
         dialog.authors = {"Ian M. Jones"};
 
         dialog.program_name = "Snippet Pixie";
-        dialog.copyright = _("Copyright © Byte Pixie Limited");
+        dialog.copyright = "Copyright © Byte Pixie Limited";
```

Comment:
This string should be translatable, even though the "Byte Pixie Limited" part of the string should remain intact.

Reply:
I thought you were talking about that too. I'm gonna fix this.